DEVELOPMENT.md

```diff
@@ -53,7 +53,7 @@ WARNING: This workflow installs additional Python packages into your Nuke's pyth
 2. Switch to your Nuke directory, like `cd "C:\Program Files\Nuke15.0v2"`.
 3. Run `.\python -m pip install -e C:\Users\<username>\deadline-clients\deadline-cloud` to install the AWS Deadline Cloud Client Library in edit mode.
 4. Run `.\python -m pip install -e C:\Users\<username>\deadline-clients\deadline-cloud-for-nuke` to install the Nuke Submitter in edit mode.
-5. Run `set NUKE_PATH=C:\Users\<username>\deadline-clients\deadline-cloud-for-nuke\src` to put the `menu.py` file in the path Nuke searches for menu extensions.
+5. Run `set NUKE_PATH=C:\Users\<username>\deadline-clients\deadline-cloud-for-nuke\src\deadline\nuke_submitter` to put the `menu.py` file in the path Nuke searches for menu extensions.
```

Comment:
Part of the reason why we wanted to include menu.py in the built wheel in the first place was for users to be able to install the integration using pip (ie. a released version)

Now that it's in the top-level of the nuke submitter it should be easy to also provide instructions on how users can do that workflow. Can we add that in this change as well?